### ISSUE TITLE
How to generate the sha3-256 hash?

### ISSUE THREAD

**Issue by yqsailor** [junkurihara/jscu#126]
I want use it to generate sha3-256 hash, what should i do?below is realization of the package [futoin-hkdf](https://github.com/futoin/util-js-hkdf#readme)

```
const hkdf = new Hkdf.Hkdf('sha3-256');
const masterPrk =hkdf.extract('aaa', 'bbb');
const secretKey =hkdf.expand(masterPrk, 'ccc', 32);
```

**Comment by junkurihara** [comment]
@yqsailor At this point, `js-crypto-hash` and other subpackages in `jscu` do not support SHA-3 series yet since `webcrypto` for browsers and `crypto` for Node.js don't as well. (`jscu` leverages these native apis to accelerate its operation.) It currently supports SHA2 series, SHA1 and MD5.

Honestly, integration of `sha3` package (https://www.npmjs.com/package/sha3) into `js-crypto-hash` seems not much tough, and I will consider to do it as a future work. (If you can, please send me a PR ;-) )

**Comment by junkurihara** [comment]
@yqsailor Hello.

Just now, I released `js-crypto-hash` sub package of v0.5.0 that supports SHA3 on Node.js and browsers. `js-crypto-hkdf`, `js-crypto-hmac`, and `js-crypto-pbkdf` supports SHA3 family as well. Now you can compute SHA3-256 hash value as follows.
```
import jschash from 'js-crypto-hash';

const computeSha3Hash =  async (msg) => {
  const hashValue = 
}
```

If you want to calculate HKDF-derived key, then please check the `README.md` in `packages/js-crypto-hkdf`.

Cheers!
